index.html

```diff
@@ -0,0 +1,539 @@
+    <div>
+      <h5>5 days ago</h5>
+      <h1>Avoid keyboard event-related box in browser-based transliteration</h1>
+      <h4>By <span>Sandamal Siripathi</span></h4>
+      <p>Transliteration is a popular way to input non-English characters using an English keyboard. This article
+        explains a new method to eliminate such browser-related problems when developing web apps that use
+        transliteration.</p>
+      <div class="article-arrow">→</div>
+    </div>
+    <div>
+      <h5>5 days ago</h5>
+      <h1>Avoid keyboard event-related box in browser-based transliteration</h1>
+      <h4>By <span>Sandamal Siripathi</span></h4>
+      <p>Transliteration is a popular way to input non-English characters using an English keyboard. This article
+        explains a new method to eliminate such browser-related problems when developing web apps that use
+        transliteration.</p>
+      <div class="article-arrow">→</div>
+    </div>
+    <div>
+      <h5>5 days ago</h5>
+      <h1>Avoid keyboard event-related box in browser-based transliteration</h1>
+      <h4>By <span>Sandamal Siripathi</span></h4>
+      <p>Transliteration is a popular way to input non-English characters using an English keyboard. This article
+        explains a new method to eliminate such browser-related problems when developing web apps that use
+        transliteration.</p>
+      <div class="article-arrow">→</div>
+    </div>
+    <div>
+      <h5>5 days ago</h5>
+      <h1>Avoid keyboard event-related box in browser-based transliteration</h1>
+      <h4>By <span>Sandamal Siripathi</span></h4>
+      <p>Transliteration is a popular way to input non-English characters using an English keyboard. This article
+        explains a new method to eliminate such browser-related problems when developing web apps that use
+        transliteration.</p>
+      <div class="article-arrow">→</div>
+    </div>
+    <div>
+      <h5>5 days ago</h5>
+      <h1>Avoid keyboard event-related box in browser-based transliteration</h1>
+      <h4>By <span>Sandamal Siripathi</span></h4>
+      <p>Transliteration is a popular way to input non-English characters using an English keyboard. This article
+        explains a new method to eliminate such browser-related problems when developing web apps that use
+        transliteration.</p>
+      <div class="article-arrow">→</div>
+    </div>
+    <div class="advert">
+      <div class="advert-div">
+        <img src="images/cat-smashing-tv-2.svg" alt="">
+        <h1>New on smashing TV</h1>
+        <p>A channel where seasoned developers show how they design, build, work and play. Coming up next: <a
+            href="#">Building A Modern Electron App With React Hooks</a> with Cassidy Williams (Mar 19).</p>
+        <h4> See all upcoming episodes →</h4>
+        <h5>Already have an account? <a href="#">Sign in!</a></h5>
+      </div>
+    </div>
+    <p class="see-all">See all latest articles →</p>
+  </article>
+  <section class="ethical">
```

Comment:
This section doesn't need to include the images and bg color. That way you can highlight it using a background color that reflects its importance.

Reply:
Done!

Reply:
Done